### DISCUSSION TITLE
我的消息界面

### DISCUSSION THREAD

**Discussion by qli5208** [ywmoyue/biliuwp-lite#466] [feature]
### 问题描述

我的消息界面跳出来的是浏览器界面，与软件整体非常割裂
<img width="981" alt="Snipaste_2024-01-01_19-24-52" src="https://github.com/ywmoyue/biliuwp-lite/assets/155362520/df610370-7e05-4ee6-bdea-e98f6d40b380">

### 描述解决方案

希望将我的消息界面改版

### 代码贡献

否

**Comment by ywmoyue** [comment]
我看了下web端好像没有使用ws保持长连接，仅仅只是请求消息数据的话做不到即时接收消息
app端使用grpc，而在uwp上用不了现成的支持http2的grpcClient库，所以这个功能大概会在日后更新框架到.net8再做
